app/scripts/controllers/provider-approval.js

```diff
@@ -22,8 +22,15 @@ class ProviderApprovalController extends SafeEventEmitter {
       providerRequests: [],
     })
 
-    const defaultState = { approvedOrigins: {} }
-    this.store = new ObservableStore(Object.assign(defaultState, initState))
+    const defaultState = {}
```

Comment:
I don't think these changes are needed anymore, since you've reverted the automatic approval of the Wyre domain.